data/synapse-data-db2/pom.xml

```diff
@@ -23,10 +23,27 @@
     <modelVersion>4.0.0</modelVersion>
     <artifactId>synapse-data-db2</artifactId>
 
+    <properties>
+        <synapse.version>0.3.30-SNAPSHOT</synapse.version>
+        <hibernate.core.version>5.6.15.Final</hibernate.core.version>
+        <hibernate.entitymanager.version>5.6.15.Final</hibernate.entitymanager.version>
```

Comment:
remove version, this should already be specify in the parent pom

Reply:
done